src/validation.cpp

```diff
@@ -1754,6 +1754,10 @@ static DisconnectResult DisconnectBlock(const CBlock& block, CValidationState& s
         }
     }
 
+    // make sure the flag is reset in case of a chain reorg
+    instantsend.isAutoLockBip9Active =
+            (VersionBitsState(pindex->pprev, Params().GetConsensus(), Consensus::DEPLOYMENT_ISAUTOLOCKS, versionbitscache) == THRESHOLD_ACTIVE);
```

Comment:
Why we use `pindex->pprev`instead of `pindex` here?

Reply:
Because after leaving this method, `pindex` is not the tip anymore but `pindex->pprev` becomes the tip.

Reply:
I see now, thanks.)